src/bootstrap/Cargo.lock

```diff
@@ -687,9 +687,9 @@ dependencies = [
 
 [[package]]
 name = "syn"
-version = "2.0.15"
+version = "2.0.8"
```

Comment:
Why downgrade? There no rustc-workspace-hack now, so no need to sync versions, plus this is bootstrap-only lock file.

Reply:
Yeah, let's try to avoid downgrading where possible. That just means another PR coming along eventually trying to bump this back up :)

Reply:
> From local testing, looks like the syn 2.0.8 -> 2.0.15 update breaks some rustc macros

Ah, missed this comment. Do we have an issue tracking that yet? It seems like that's probably breakage that needs to be reported upstream to syn?

Reply:
This is bootstrap lock, and breakage with version bump in workspace lock, i assume.